x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/pipelines/ml_inference/configure_pipeline.tsx

```diff
@@ -113,222 +107,162 @@ export const ConfigurePipeline: React.FC = () => {
 
   const inputsDisabled = configuration.existingPipeline !== false;
 
-  return (
-    <>
-      <EuiFlexGroup>
-        <EuiFlexItem grow={3}>
-          <EuiTitle size="s">
-            <h4>
-              {i18n.translate(
-                'xpack.enterpriseSearch.content.indices.pipelines.addInferencePipelineModal.steps.configure.title',
-                { defaultMessage: 'Create or select a pipeline' }
-              )}
-            </h4>
-          </EuiTitle>
+  const tabs: EuiTabbedContentTab[] = [
+    {
+      content: (
+        <>
```

Comment:
In JSX there is always a single top level component. So each return statement or property is either a component or a `<>` that wraps multiple sibling components.

Reply:
So that the JSX expression returned has a single parent element. See here: https://kinsta.com/knowledgebase/jsx-expressions-must-have-one-parent-element/